### PR TITLE
UHM-5561 Patient chart in CI environments should link to MFE patient …

### PR DESCRIPTION
**Pull request by brandones**
…chart

https://pihemr.atlassian.net/browse/UHM-5561

Depends on https://github.com/PIH/openmrs-module-pihcore/pull/280

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -434,6 +434,10 @@ private void loadAppsAndExtensions() throws UnsupportedEncodingException {
             enableCommentForm();
         }
 
+//        if (config.isComponentEnabled(Components.SPA_PREVIEW)) {
```
Comment (mseaton) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java:437]: Why is this commented out?

**Review by mseaton — APPROVED** [review]
Sure, looks fine, but one comment about commented out code

**Review by mogoodrich — CHANGES_REQUESTED** [review]
Looks like the line to make sure SPA preview is only conditionally enabled is mistakenly commented out?

**Review by mseaton — APPROVED** [review]

**Review by mogoodrich — APPROVED** [review]
LGTM, thanks @brandones !

**Pull request merged**